Source/Core/Core/PowerPC/Jit64/Jit_LoadStore.cpp

```diff
@@ -315,7 +315,10 @@ void Jit64::dcbz(UGeckoInstruction inst)
 	MOV(32, R(RSCRATCH), gpr.R(b));
 	if (a)
 		ADD(32, R(RSCRATCH), gpr.R(a));
-	AND(32, R(RSCRATCH), Imm32(~31));
+	if (SConfig::GetInstance().m_LocalCoreStartupParameter.bWii)
+		AND(32, R(RSCRATCH), Imm32(~31));
+	else
+		AND(32, R(RSCRATCH), Imm32(Memory::RAM_MASK & ~31));
```

Comment:
The addresses that the game tries to DCBZ are 0x026001a0 and 0x026001c0.

PR #1856 doesn't fix the crash.  PR #1882 fixes the crash if the MMU option is disabled and crashes if the MMU option is enabled.  There are no pop-ups with "Bad BAT setup".

What do you suggest?  I suppose I could RAM_MASK the address in the fast path before the PXOR, if the game is a GC game.  Otherwise, we'd have to check the first nibble of the address.